Webpack and Babel Task-2/src/App.js

```diff
@@ -0,0 +1,14 @@
+import React, { useState, Suspense } from 'react';
+//import Headlines from './components/Headlines';
+const Headlines = React.lazy(() => import('./components/Headlines'));  //lazy loading
+
+export default function App() {
+    const [state, setState] = useState(false);
```

Comment:
Try not using state as variable name, use meaningful name like isHeadingVisible/showHeading and
instead of SetState you can use like toggleHeadlines

Reply:
> Try not using state and setState as variable name, use meaningful name like isHeadingVisible/showHeading and instead of SetState you can use like toggleHeadlines

done